test.config.js

```diff
@@ -26,9 +26,22 @@ Enzyme.configure({
   adapter: new Adapter(),
 })
 
+try {
+  window.matchMedia =
+    window.matchMedia ||
+    function() {
+      return {
+        matches: false,
+        addListener: function() {},
+        removeListener: function() {},
+      }
+    }
+} catch (error) {}
```

Comment:
This has been destroying ci lately too, i wonder why its just now happening? Seems so random.

Reply:
what do you mean by destroy?  I know we use `matchMedia` in alot of publishing components from reaction, added this so we don't have to mock it in tests.

Reply:
Haha sorry sorry, I mean like false-positive failures that a rerun will fix.

Reply:
Cool hopefully this will kill any of those failures.

Reply:
:( Just had this error in a PR after pulling latest (including this change)

```
/home/circleci/project/node_modules/enquire.js/src/MediaQueryDispatch.js:15
        throw new Error('matchMedia not present, legacy browsers require a polyfill');
        ^

Error: matchMedia not present, legacy browsers require a polyfill
```

It's weird that I don't recall _ever_ seeing this error before..like a week ago? But it's also hard to tell where it's coming from. Here's that failure: https://circleci.com/gh/artsy/force/16564#tests/containers/1

Luckily (🤞 ) that does seem to _usually_ pass on the next retry (vs several retries).

Reply:
Do we know which test file is triggering this? I can see the error, but not the test where it fails in CI.

Reply:
Same, couldn't tell.

Reply:
This test config file should only apply to Jest, so I think it is at least safe to say it is failing in a mocha test.

Reply:
Currently we use it for both: 

<img width="234" alt="screen shot 2019-02-22 at 11 01 11 am" src="https://user-images.githubusercontent.com/236943/53264739-32af0700-3691-11e9-901f-ebddb0959766.png">

Reply:
Ah I see -- my mistake.  I did notice that trying to declare some globals in this file had unintended consequences in mocha.  For example, adding query as a global would break `benv`'s ability to use jquery.  Perhaps we need to explicitly add `matchMedia` to some tests via `benv`. I am guessing this comes from the responsive components in Reaction/Palette.